man/systemd-analyze.xml

```diff
@@ -792,6 +792,16 @@ Service b@0.service not loaded, b.socket cannot be started.
         as well and its default value is 100.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--json-path=<replaceable>PATH</replaceable></option></term>
```

Comment:
json is an implementation detail. The switch should not document that in its name, but what this is for, i.e. be named "--security-policy=" or "--security-custom=" or "--custom-policy=" or so.